README.md

```diff
@@ -105,7 +105,7 @@ Once you have these:
     # Install the project dependencies
     $ gem install bundler
     $ bundle install
-    $ yarn install
+    $ yarn install [using version 9 of the node](https://github.com/creationix/nvm#deeper-shell-integration)
```

Comment:
We don't need the "the" before pronouns like that. What do you think about changing it to:
```suggestion
    $ yarn install (while using Node version 9. Check [this link](https://github.com/creationix/nvm#deeper-shell-integration) if you use NVM)
```